recipes/gdcm/all/patches/0001-3.0.20-find-charls.patch

```diff
@@ -0,0 +1,28 @@
+Allow using charls >= 2.2.0
+https://github.com/malaterre/GDCM/pull/157
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -352,8 +352,8 @@ else()
+ endif()
+ 
+ if(GDCM_USE_SYSTEM_CHARLS)
+-  find_package(CharLS 2.0.0 REQUIRED)
+-  set(GDCM_CHARLS_LIBRARIES ${CHARLS_LIBRARIES})
++  find_package(charls 2.2.0 REQUIRED)
```

Comment:
But required version in ``conanfile.py`` is ``charls/2.4.2``, won't this be a problem?

Reply:
I hope that CMakeToolchain/CMakeDeps always look for version that conan provides. So any value here should be ok.

Reply:
It looks for compatible version, so 2.6.2 fits. Only when `EXACT` is present in `find_package` CMake will requires only that specific version.

https://cmake.org/cmake/help/v3.15/command/find_package.html